packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -506,12 +511,22 @@ abstract class TestRunner {
       remoteStoreOnlineStateChangedHandler,
       connectivityMonitor
     );
-    this.syncEngine = new SyncEngine(
-      this.localStore,
-      this.remoteStore,
-      this.sharedClientState,
-      this.user
-    );
+    if (this.maxConcurrentLimboResolutions) {
```

Comment:
If there's more configuration knobs like this one, this kind of separate invocation for each configuration will explode in complexity. It's probably better to have a named constant, export it, and just use that default value here.

Alternatively, maybe set the default here to something obscenely huge to prevent limbo throttling altogether if it's not intended?

Reply:
Agreed.  Thanks for pointing this out because I didn't like it either.  I've changed `maxConcurrentLimboResolutions` to a required parameter, delegating the responsibility to the caller to determine the value.  I then updated `firestore_client.ts` to specify the value of `100` and used `Number.MAX_SAFE_INTEGER` in the spec tests, unless explicitly overridden by a specific test.